js/idex.js

```diff
@@ -56,8 +56,8 @@ module.exports = class idex extends Exchange {
                 },
                 'logo': 'https://user-images.githubusercontent.com/51840849/94481303-2f222100-01e0-11eb-97dd-bc14c5943a86.jpg',
                 'api': {
-                    'public': 'https://api.idex.io',
-                    'private': 'https://api.idex.io',
+                    'ETH': 'https://api-eth.idex.io',
+                    'BSC': 'https://api-bsc.idex.io/',
```

Comment:
@frosty00 it will end up with a double slash here, is that intentional?

Reply:
uh, it just worked when I tested it

Reply:
apparently it works with two slashes